aws/aws-explanation/eks-snaps.rst

```diff
@@ -0,0 +1,26 @@
+Snap Usage in EKS Worker Images
+===============================
+
+The EKS worker images bundle several snaps that provide
+kubernetes components. This page will provide a brief
+explanation on what snaps are bundled and why.
+
+Snaps
+=====
+
+.. list-table::
+   :header-rows: 1
+
+   * - **Snap Name**
+     - **Provides**
+     - **Bundled Since**
+   * - kubelet-eks
```

Comment:
Can you link to https://documentation.ubuntu.com/aws/en/latest/aws-reference/eks-kubelet-snap/ ?

Reply:
waiting on rebuild to make sure the relative link works properly, but done